Intermittent DatabaseDriverTmpfsTest::tmpfs test failures in CI

See Azure CI reports for a summary: https://dev.azure.com/testcontainers/testcontainers-java/_test/analytics?definitionId=5&contextType=build - this test has a ~94% pass rate.

We should investigate why this is a flaky test and resolve.

Error message:
```
java.lang.AssertionError: tmpfs inside container works fine: 'ls: cannot access '/testtmpfs/test.file': No such file or directory
' does not equal expected 'ls: 
cannot access '/testtmpfs/test.file': No such file or directory
'
```

--- a/modules/jdbc-test/src/test/java/org/testcontainers/jdbc/DatabaseDriverTmpfsTest.java
+++ b/modules/jdbc-test/src/test/java/org/testcontainers/jdbc/DatabaseDriverTmpfsTest.java
@@ -1,27 +1,19 @@
 package org.testcontainers.jdbc;
 
-import org.junit.Rule;
 import org.junit.Test;
 import org.testcontainers.containers.Container;
 import org.testcontainers.containers.JdbcDatabaseContainer;
-import org.testcontainers.testsupport.Flaky;
-import org.testcontainers.testsupport.FlakyTestJUnit4RetryRule;
 
-import java.io.IOException;
 import java.sql.Connection;
 import java.sql.DriverManager;
-import java.sql.SQLException;
 
 import static org.rnorth.visibleassertions.VisibleAssertions.assertEquals;
+import static org.rnorth.visibleassertions.VisibleAssertions.assertNotEquals;
 
 public class DatabaseDriverTmpfsTest {
 
-    @Rule
-    public FlakyTestJUnit4RetryRule retry = new FlakyTestJUnit4RetryRule();
-
     @Test
-    @Flaky(githubIssueUrl = "https://github.com/testcontainers/testcontainers-java/issues/1687", reviewDate = "2020-03-01")
-    public void tmpfs() throws IOException, InterruptedException, SQLException {
+    public void testDatabaseHasTmpFsViaConnectionString() throws Exception {
         final String jdbcUrl = "jdbc:tc:postgresql:9.6.8://hostname/databasename?TC_TMPFS=/testtmpfs:rw";
         try (Connection ignored = DriverManager.getConnection(jdbcUrl)) {
 
@@ -29,15 +21,11 @@ public class DatabaseDriverTmpfsTest {
             // check file doesn't exist
             String path = "/testtmpfs/test.file";
             Container.ExecResult execResult = container.execInContainer("ls", path);
-            assertEquals("tmpfs inside container works fine", execResult.getStderr(),
-                "ls: cannot access '/testtmpfs/test.file': No such file or directory\n");
+            assertNotEquals("tmpfs inside container doesn't have file that doesn't exist", 0, execResult.getExitCode());
             // touch && check file does exist
             container.execInContainer("touch", path);
             execResult = container.execInContainer("ls", path);
-            assertEquals("tmpfs inside container works fine", execResult.getStdout(), path + "\n");
+            assertEquals("tmpfs inside container has file that does exist", 0, execResult.getExitCode());
         }
-
     }
-
-
 }

